Makefile

```diff
@@ -5,18 +5,28 @@ VERSION := $(shell yq e ".version" manifest.yaml)
 
 all: verify
 
+clean:
+	rm -f thunderhub.s9pk
+	rm -f image.tar
+	rm -f Dockerfile*
+	rm -f scripts/*.js
+
 verify: thunderhub.s9pk
 	embassy-sdk verify s9pk thunderhub.s9pk
 
 install: thunderhub.s9pk
 	embassy-cli package install thunderhub.s9pk
 
-thunderhub.s9pk: manifest.yaml assets/compat/config_spec.yaml assets/compat/config_rules.yaml image.tar instructions.md
+thunderhub.s9pk: manifest.yaml assets/compat/config_spec.yaml assets/compat/config_rules.yaml image.tar instructions.md scripts/embassy.js 
 	embassy-sdk pack
 
 Dockerfile: $(THUNDERHUB_SRC)
-	cp thunderhub/arm64v8.Dockerfile Dockerfile
+	cp thunderhub/Dockerfile Dockerfile
```

Comment:
umm this may not be an allowed change. Are we sure this is always arm64 compatible

Reply:
the new update to thunderhub included multiarch. the new Dockerfile is now compatible with arm64

Reply:
kk